tests/plugins/preview/manual/previewcdn.html

```diff
@@ -0,0 +1,19 @@
+<iframe width="800px" height="400px" id="cdn"></iframe>
+
+<script>
+	// Encapsulate environment inside iframe.
+	// It's prevent to open local version of CKEDITOR by bender.
+	var iframe = document.getElementById('cdn'),
+		doc = iframe.contentWindow.document,
+		template = '<h1>Sample <span style="color: red;">document</span></h1>' +
+			'<p>Lorem ipsum</p>' +
+			'<div style="page-break-after:always"><span style="display:none">&nbsp;</span></div>' +
+			'<p>dolor sit amet.</p>';
+
+	doc.open();
+	// CDN points into commit revision with bug fix.
+	doc.write( '<script src="https://raw.githack.com/ckeditor/ckeditor4/be63bc46be15478259dfbca94de6a019b0b0145c/ckeditor.js"><\/script>' );
```

Comment:
I'm not happy with this solution. It requires updating the commit hash on every change inside the PR.

Yet I don't see a better solution :thinking: